src/main/java/svnserver/ext/gitlfs/server/LfsContentManager.java

```diff
@@ -118,7 +118,11 @@ User checkDownload(@NotNull HttpServletRequest request) throws IOException, Unau
   private User checkAccess(@NotNull HttpServletRequest request, @NotNull Checker checker) throws IOException, UnauthorizedError, ForbiddenError {
     final User user = getAuthInfo(request);
     try {
-      checker.check(user, "/");
+      // This is a *bit* of a hack.
+      // If user accesses LFS, it means she is using Git. If she uses git, she has whole repository contents.
+      // If she has full repository contents, it doesn't make sense to apply path-based authorization.
+      // Setups where where user has Git access but is not allowed to write via path-based authorization are declared bogus.
```

Comment:
It should better be mentioned in the documentation though.

Reply:
makes sense